### PR TITLE
add lesson completed action hook

### PR DESCRIPTION
**Pull request by octaviorequena**
Source: [NTGR-716](https://accredible.atlassian.net/browse/NTGR-716)

### CODE REVIEW

File: includes/class-accredible-learndash-event-handler.php [hunk 1]
```diff
@@ -46,6 +46,34 @@ public static function handle_course_completed( $data ) {
 			return count( $auto_issuances );
 		}
 
+		/**
+		 * Handle `learndash_lesson_completed` Action Hooks
+		 *
+		 * @param Array $data lesson data.
+		 */
+		public static function handle_lesson_completed( $data ) {
```
Comment (kei178) [inline includes/class-accredible-learndash-event-handler.php:54]: Would it be possible to extract the common logic between `handle_course_completed` and `handle_lesson_completed` to some private function for DRY?

**Review by kei178 — CHANGES_REQUESTED** [review]
@octaviorequena - I left a comment on refactoring.

File: includes/class-accredible-learndash-event-handler.php [hunk 2]
```diff
@@ -52,24 +35,36 @@ public static function handle_course_completed( $data ) {
...
+
+			return $auto_issuances_count;
+		}
+
+		/**
+		 * Find auto issuance and create credential when a course/lesson is completed by a student.
+		 *
+		 * @param Array $kind Auto issuance kind.
+		 * @param Array $post_id Auto issuance post ID.
+		 * @param Array $user user data.
```
Comment (kei178) [inline includes/class-accredible-learndash-event-handler.php:48]: Data types are incorrect.

**Review by kei178 — APPROVED** [review]
Looks good. I left just one minor comment on the PHP doc.

File: includes/class-accredible-learndash-event-handler.php (hunk 2 above)
Comment (octaviorequena) [reply includes/class-accredible-learndash-event-handler.php]: Fixed!

**Review by Ndibui — APPROVED** [review]

**Pull request merged**